lib/chromic_pdf/template.ex

```diff
@@ -202,6 +193,54 @@ defmodule ChromicPDF.Template do
   def html_concat(styles, {:safe, content}), do: html_concat(styles, content)
   def html_concat(styles, content), do: [styles, content]
 
+  @doc """
+  Returns an options list for given template options.
+
+  Returned options can be passed as second argument to `ChromicPDF.print_to_pdf/2`.
+
+  ## Options
+
+  * `header` iodata content of header
+  * `footer` iodata content of footer
+  * all options from `styles/1`
+
+  ## Example
+
+  This example has the dimension of a ISO A4 page.
+
+      ChromicPDF.Template.options(
+        header: "<p>header</p>",
+        footer: "<p>footer</p>",
+        size: :a4,
+        header_height: "45mm",
+        header_font_size: "20pt",
+        footer_height: "40mm"
+      )
+
+  Header, and footer templates should be unwrapped HTML markup (i.e. no `<html>` around
+  the content), including any `<style>` tags that your page needs.
+  """
+  @spec options() :: keyword()
+  @spec options([header_footer_option() | style_option() | {:styles, binary()}]) :: keyword()
+  def options(opts \\ []) do
+    {width, height} = get_paper_size(opts)
+
+    header = Keyword.get(opts, :header, "")
+    footer = Keyword.get(opts, :footer, "")
+    styles = Keyword.get_lazy(opts, :styles, fn -> styles({width, height}, opts) end)
+
+    [
+      print_to_pdf: %{
+        displayHeaderFooter: true,
+        preferCSSPageSize: true,
+        headerTemplate: html_concat(styles, header),
+        footerTemplate: html_concat(styles, footer),
+        paperWidth: width,
+        paperHeight: height
+      }
+    ]
+  end
```

Comment:
Maybe it could be nice to add a test for the `options` function to make sure that we can in fact pass that in to `print_to_pdf`.